pages/prelude/terra-box/index.md

```diff
@@ -3,26 +3,97 @@ title: Terra-Box
 permalink: /terra-box_index.html
 datatable: false
 toc: true
+serviceImage: terra-box/logo.png
 ---
 
-Terra-Box is a [HCL](https://github.com/hashicorp/hcl) based project that defines the full WDP infrastructure to be deployed on Kubernetes.
+Terra-Box is a [HCL](https://github.com/hashicorp/hcl) based project that defines the full WDP infrastructure for deploying it on Kubernetes.
 
 This page describes how the project is organized and explains a few of concepts that you should understand to work with WDP deployments.
 
-## Project structure
+## Terra-Box Definition Tree (TDT) {#tdt}
 
-Terra-box is organized in two main parts:
+WDP is composed by several microservices or components which work together. On Terra-Box, a set of components is defined by a "Terra-Box Definition Tree" (TDT). It is a way of organizing the Terraform files which defines how a component is deployed on Kubernetes. A TDT is organized on the following folders:
 
-* **Base**: It includes the definition of the main components WDP.
+* modules
+* main
+* templates
 
-* **Platforms**: It defines components and configurations which add support to deploy WDP in many cloud providers (AWS, GCP, Azure, ...) or on-premise environments.
+{% include image.html file="terra-box/Terra-BoxTDT.png" alt="Terra-Box TDT" max-width=500 %}
 
-## Modules for WDP components
+### modules
 
-## Main modules
+This is the place where the WDP components are defined. Each component definition is created as a Terraform module and it can be imported and used by a higher level module (defined in "main").
 
-## Templates
+For instance, Wizz-Vis is a WDP component defined in `modules`. It has two Terraform files:
 
-## Overriding modules
+* `main.tf` where the kubernetes resources which deploys Wizz-Vis are defined. This module creates several kubernetes [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment), a [service](https://kubernetes.io/docs/concepts/services-networking/service/) and a [secret](https://kubernetes.io/docs/concepts/configuration/secret).
 
-## ...
+* `vars.tf` where the module [Input Variables](https://learn.hashicorp.com/terraform/getting-started/variables) (commonly simply called as **vars**) are declared. Module vars let us to parametrize configuration details of the component. E.g.: The password of the admin user.
+
+Most of this components can be deployed on kubernetes in a isolated way but some of them might have dependencies.
```

Comment:
Terrabox is not designed to deploy modules independently, so I would avoid this phrase